templates/README.md

```
@@ -130,6 +130,9 @@ Credit: https://github.com/ripta/dotfiles/tree/master/zsh-custom/plugins/kube/te
 - `crds.tmpl` - Custom Resource Definitions with conversion strategy
 - `finalizers.tmpl` - Resources with finalizers blocking deletion
 
+#### Karpenter
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)